### PR TITLE
sync changes with skynet-webportal website package

### PR DESCRIPTION
**Pull request by kwypchlo**
- copied files from https://github.com/SkynetLabs/skynet-webportal/tree/master/packages/website
- mostly just Dockerfile changes
- this repo should just be a fork honestly... todo when website is extracted to its own repo (cc @MSevey)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 WORKDIR /usr/app
 
 # disable gatsby telemetry and installing cypress binary
 ENV GATSBY_TELEMETRY_DISABLED 1
 ENV CYPRESS_INSTALL_BINARY 0
 
-COPY package.json yarn.lock ./
+COPY package.json \
+     yarn.lock \
+     ./
```
Comment (ro-tex) [inline Dockerfile:13]: I see we're no longer running `gatsby build`. Is that intended?
If so, should we clean it up from `package.json`?

**Review by ro-tex — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (kwypchlo) [reply Dockerfile:13]: We are running it in the CMD section

https://github.com/SkynetLabs/skynet-websites/pull/12/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R26

Comment (ro-tex) [reply Dockerfile:13]: Oh, right. Hm. Is this a good idea? I know that it works but it seems so wrong. If the idea is that we don't want to create and additional layer we can always do it together with `yarn --frozen-lockfile` and make sure we do it after we do all the copying.

Comment (kwypchlo) [reply Dockerfile:13]: We need to build in the context of the env vars, otherwise we can't customise the metadata that ends up in the html.

**Pull request merged**